docs/controls/src/imageviewer-desktop/main.qml

```diff
@@ -0,0 +1,86 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.4
+import QtQuick.Dialogs 1.2
```

Comment:
I wonder if we should use the dialogs from Qt.labs.platform instead?

Reply:
I wanted a pure QML solution first, since the labs module requires QtWidgets. I've added a note about this, and there is a separate section on the labs module.